pennylane/__init__.py

```diff
@@ -133,6 +136,63 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, name: str):
+    """load(name, quantum_circuit_object)
+    Load a plugin :func:`~.load` which can then be used to convert objects of
```

Comment:
Best to write this in a way that makes sense to a user reading the documentation:
```suggestion
    Load external quantum assembly and quantum circuits from supported frameworks
    into PennyLane templates.
```